examples/speaker_tasks/diarization/offline_diarization_with_asr.py

```diff
@@ -47,35 +47,25 @@ def main(cfg):
 
     logging.info(f'Hydra config: {OmegaConf.to_yaml(cfg)}')
 
-    asr_diar_offline = ASR_DIAR_OFFLINE(**cfg.diarizer.asr.parameters)
-    asr_diar_offline.root_path = cfg.diarizer.out_dir
-    shutil.rmtree(asr_diar_offline.root_path, ignore_errors=True)
+    # ASR inference for words and word timestamps
+    asr_ts_decoder = ASR_TIMESTAMPS(**cfg.diarizer)
+    asr_model = asr_ts_decoder.set_asr_model(cfg.diarizer.asr.model_path)
```

Comment:
minor: ASR_model_name is already being assigned during the initialization of ASR_TIMESTAMPS. You may call this without arg and use self.ASR_model_name directly in function.

Reply:
Need to discuss this.